numerics/root_finders.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace principia {
+namespace numerics {
+
+// Approximates a root of |f| between |lower_bound| and |upper_bound| by
```

Comment:
Mention that lower_ and upper_bound must have values of different signs and the function must be monotonic.